hathor/feature_activation/exception.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+class FeatureActivationException(Exception):
+    pass
+
+
+class InvalidFeaturesConfigurationException(FeatureActivationException):
```

Comment:
I'd say it should stop the full node completely. So, I feel we should add a validation step at the beginning of the full node initialization (and maybe this exception should inherit from a `InitializationFailed` or `PreInitializationFailed` exception).

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/611/commits/b141815b3655383aa17f7df5c33cba323cb58c19

Reply:
Great! Thanks!